flytekit/tools/fast_registration.py

```diff
@@ -139,9 +141,21 @@ def fast_package(
                         filter=lambda x: tar_strip_file_attributes(x),
                     )
 
-            with gzip.GzipFile(filename=archive_fname, mode="wb", mtime=0) as gzipped:
-                with open(tar_path, "rb") as tar_file:
-                    gzipped.write(tar_file.read())
+            if shutil.which("pigz"):
+                with open(archive_fname, "wb") as gzipped:
+                    subprocess.run(["pigz", "-c", tar_path], stdout=gzipped)
+            else:
+                start_time = time.time()
+                with gzip.GzipFile(filename=archive_fname, mode="wb", mtime=0) as gzipped:
+                    with open(tar_path, "rb") as tar_file:
+                        gzipped.write(tar_file.read())
+
+                end_time = time.time()
+                warning_time = 30
+                if end_time - start_time > warning_time:
+                    logger.info(
```

Comment:
Should we use `click.secho` instead? we don't print `logger.info` by default

Reply:
https://github.com/flyteorg/flytekit/pull/2729/commits/1f8018134616fb97d47e5436665131eb581c8d43